general/development/policies/codingstyle/index.md

```diff
@@ -1520,7 +1520,7 @@ There are some tags that are only allowed within some contexts and not globally.
 - `@covers`, `@coversDefaultClass`, `@coversNothing`, `@uses` to better control coverage within [unit tests](https://docs.moodle.org/dev/Writing_PHPUnit_tests#Generators).
 - `@dataProvider` and `@testWith`, to provide example data and expectations, within [unit tests](https://docs.moodle.org/dev/Writing_PHPUnit_tests#Generators).
 - `@depends`, to express dependencies between tests, where each producer returned data in passed to consumers. See [`@depends` examples](https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#writing-tests-for-phpunit-examples-stacktest2-php) for more information.
-- `@group`, for easier collecting unit tests together, following the guidelines in the [PHPUnit MoodleDocs](https://docs.moodle.org/dev/PHPUnit#Using_the_.40group_annotation).
+- `@group`, for easier collecting unit tests together, following the guidelines in the [PHPUnit MoodleDocs](/general/development/tools/phpunit#using-the-.40group-annotation).
```

Comment:
Just need to double-check the bookmark link here. The migration tool attempts to fix this, but they're not easy to get right and this one looks wrong.

Reply:
Thanks Andrew! Fixed